tests/public-api/Makefile

```diff
@@ -49,3 +49,18 @@ test-ci: ## runs ci tests (i.e. w/o debug tools and with code coverage)
 	--color=yes \
 	--durations=10 \
 	$(CURDIR)
+
+.PHONY: install-latest-python-osparc
+install-latest-python-osparc: ## install the latest python osparc package (the latest built wheels on ITISFoundation/osparc-simcore-clients master)
+	$(eval REPO := ITISFoundation/osparc-simcore-clients)
+	$(eval BRANCH := master)
+	$(eval WORKFLOW := publish-and-test-python-client)
```

Comment:
Q: why you do not declare these variables in the Makefile?

Reply:
Where else should I put them? in `scripts/common.Makefile`?

Reply:
no I meant like this:
```
REPO := ITISFoundation/osparc-simcore-clients
BRANCH := master
WORKFLOW := publish-and-test-python-client
.PHONY: install-latest-python-osparc
install-latest-python-osparc: _check_venv_active ## install the latest python osparc package (the latest built wheels on ITISFoundation/osparc-simcore-clients master)
```

Reply:
but that is a minor thing. was just wondering

Reply:
Ah, right. That's a good point. I will do fix that. Thanks for pointing this out.